docs/sdk/tutorials/export_a_kili_project.md

```diff
@@ -91,6 +95,42 @@ for asset in assets:
                 break # once we find a latest label done by a reviewer, we move on to the next asset.
 ```
 
+### Filtering the latest label per annotator
+When working on project with the [consensus](https://docs.kili-technology.com/docs/consensus-overview) activated, it can be worth it export the latest label done by each annotator:
```

Comment:
When working on a project with consensus enabled, it can be useful to export the latest label made by each annotator